contracts/facets/ArchaeologistFacet.sol

```diff
@@ -11,7 +11,7 @@ import {AppStorage} from "../storage/LibAppStorage.sol";
 contract ArchaeologistFacet {
     AppStorage internal s;
 
-    event UnwrapSarcophagus(bytes32 indexed sarcoId, bytes unencryptedShard);
+    event PublishKeyShare(bytes32 indexed sarcoId, bytes rawKeyShare);
```

Comment:
Renaming all instances of `unwrap` to `publishKeyShare` to make it more clear that archaeologists aren't decrypting the sarcophagus, they're just supplying the share they're responsible for

Reply:
I agree with using the term "unencrypted" since that's how we have it throughout the entire app.

Reply:
Are we going with using the word "share" globally instead of "shard". Technically is should probably be "share" but we have "shard" all over the place.

Reply:
In the contracts what we're really trying to differentiate between is the doubleHash of the keyShare and the unprocessed (raw) keyShare, since there's no encrypted key share anywhere. 

I can understand the distinction between encrypted and unencrypted being a lot more important client side though and I think this is a scenario where it's appropriate for the webapp to use `unencryptedKeyShare` because the webapp will have `encryptedKeyShare`s as well. 

As for shares vs shards I think we ought to standardize on shares across all codebases and documentation (and definitely in the UI) to avoid any confusion.